tests/tests_default.yml

```diff
@@ -2,6 +2,38 @@
 ---
 - name: Ensure that the role runs with default parameters
   hosts: all
-  gather_facts: false  # test that role works in this case
+  become: true
   roles:
-    - linux-system-roles.template
+    - linux-system-roles.journald
+
+  tasks:
+    - name: Get systemd-journald service state
+      command:
+        cmd: systemctl show --property SubState systemd-journald.service
+      register: systemctl
+      changed_when: false
+
+    - name: Verify that systemd-journald is running
+      assert:
+        that: systemctl.stdout == 'SubState=running'
+
+    - name: Verify that /run/log/journal/$MACHINE_ID/system.journal exists
+      command:
+        cmd: stat /run/log/journal/{{ ansible_machine_id }}/system.journal
+      changed_when: false
+
+    - name: Verify that system.journal is in online state
+      shell:
+        cmd: >
+          "journalctl --header
+          --file /run/log/journal/{{ ansible_machine_id }}/system.journal |
```

Comment:
```suggestion
          --file /run/log/journal/{{ ansible_machine_id }}/system.journal | \
```

Reply:
Is the continuation backslash necessary? I want the entire command to be executed as if it was written on one line.

Reply:
Sorry, I thought it was using the yaml string `|` operator - the `>` operator is fine